### PR TITLE
Fix commit message examples to use imperative mood

### LINKED ISSUE #972: book v2.1.33 errata
[book/05-distributed-git/sections/contributing.asc](https://github.com/progit/progit2/blob/master/book/05-distributed-git/sections/contributing.asc)
> It’s also a good idea to use the imperative present tense in these messages. In other words, use commands. Instead of “I added tests for” or “Adding tests for,” use “Add tests for.” Here is a template originally written by Tim Pope:

The book do not always follow its own recommondation about proper git commit messages.

The secction "Committing Your Changes" should point a reference to this recommondation section.

Errata:
[book/02-git-basics/sections/viewing-history.asc](https://github.com/progit/progit2/blob/master/book/02-git-basics/sections/viewing-history.asc)
     changed the version number => Change the version number
     removed unnecessary test => Remove unnecessary test
 
[book/03-git-branching/sections/nutshell.asc](https://github.com/progit/progit2/blob/master/book/03-git-branching/sections/nutshell.asc)
  Fixed bug #1328 => Fix bug #1328
  made a change => make a change
  made other changes => make other changes

There are different case in two examples:
34ac2 Fixed bug #1328
34ac2 fixed bug #1328

[book/03-git-branching/sections/basic-branching-and-merging.asc](https://github.com/progit/progit2/blob/master/book/03-git-branching/sections/basic-branching-and-merging.asc)
 added a new footer => add a new footer
 finished the new footer => finish the new footer
 fixed the broken email address => fix the broken email address
 
[book/03-git-branching/sections/remote-branches.asc](https://github.com/progit/progit2/blob/master/book/03-git-branching/sections/remote-branches.asc)
 forgot the brackets =>  add forgotten brackets
 deploying index fix => deploy index fix
 trying something new => try something new
 
[book/05-distributed-git/sections/contributing.asc](https://github.com/progit/progit2/blob/master/book/05-distributed-git/sections/contributing.asc)
 made the ls-tree function recursive => make the ls-tree function recursive
 
[book/05-distributed-git/sections/maintaining.asc](https://github.com/progit/progit2/blob/master/book/05-distributed-git/sections/maintaining.asc)
 seeing if this helps the gem => see if this helps the gem
 updated the gemspec to hopefully work better => update the gemspec to hopefully work better
 
This is all I have for now after gotten to page 153 (PDF v2.1.33).

### LINKED ISSUE #1271: Commit message examples in book not using imperative mood
**Which version of the book is affected?**

Problem is present in source files. So naturally all that follows from the source files is also affected.

**Describe the bug:**

Not all example commit messages in the source files are using the imperative mood.

**Steps to reproduce:**

1. Go to https://github.com/progit/progit2/blob/master/book/03-git-branching/sections/basic-branching-and-merging.asc
and search for: "fixed the broken email address".
2. Commit message is not written in the imperative mood (the way git talks to you on the command line).

**Expected behavior:**

I think it would set a good standard if the readers get to see "proper" commit messages throughout the book. This way the readers will get familiar with the correct way to write commit messages.

**Screenshots:**

**Additional context:**

Reference to our style guide for commits:
From https://github.com/progit/progit2/blob/master/book/05-distributed-git/sections/contributing.asc
Write your commit message in the imperative: "Fix bug" and not "Fixed bug" or "Fixes bug."

I'm willing to fix the issue, by making new pull-requests, if @ben wants to change the example commit messages too.

Related issue #972.

### PR DESCRIPTION
**Pull request by HonkingGoose**
Closes #1271
Closes #972

### CODE REVIEW

File: book/05-distributed-git/sections/contributing.asc [hunk 1]
```diff
@@ -90,14 +90,6 @@ Further paragraphs come after blank lines.
...
 The Git project has well-formatted commit messages -- try running `git log --no-merges` there to see what a nicely-formatted project-commit history looks like.
 
-[NOTE]
-.Do as we say, not as we do.
-====
-For the sake of brevity, many of the examples in this book don't have nicely-formatted commit messages like this; instead, we simply use the `-m` option to `git commit`.
-
-In short, do as we say, not as we do.
-====
-
```
Comment (ben) [inline book/05-distributed-git/sections/contributing.asc]: Let's actually keep this, since we're still using the `-m` flag all over the place.

Comment (HonkingGoose) [reply book/05-distributed-git/sections/contributing.asc]: Hmm that took a bit of work to figure out how to get that bit out of the commit... 😄 

Managed to do it with:
1. `git rebase -i upstream/master`, selecting `edit` on the commit.
2. First stage all the known good stuff with git add.
3. `git add -p book/05-distributed-git/sections/contributing.asc` to stage the good stuff from that section.
4. `git restore book/05-distributed-git/sections/contributing.asc` to leave the single bad part behind.

**Comment by HonkingGoose** [comment]
Hmm, I just noticed that some pictures are using the old style commit messages.

Examples:
- https://github.com/progit/progit2/blob/master/images/mentions-02-render.png
- https://github.com/progit/progit2/blob/master/images/blink-02-pr.png
- https://github.com/progit/progit2/blob/master/images/blink-03-pull-request-open.png
- https://github.com/progit/progit2/blob/master/images/blink-05-general-comment.png
- https://github.com/progit/progit2/blob/master/images/blink-06-final.png
- https://github.com/progit/progit2/blob/master/images/commit-and-tree.png
- https://github.com/progit/progit2/blob/master/images/commits-and-parents.png

This is something to think about before merging this pull-request.

**Comment by lazarljubenovic** [comment]
I just want to say that I'm amazed by the dedication! 💚

**Comment by HonkingGoose** [comment]
@lazarljubenovic Thank you! 😄

**Comment by ben** [comment]
❤️

**Comment by HonkingGoose** [comment]
Thanks for merging my work! 😄 ✨
